_posts/2016-02-27-write-code-for-the-dumbest-person-you-know.md

```diff
@@ -20,15 +20,15 @@ You're lucky. You're lucky because you're clever and I'm not.
 I noted that I was dumb when I was unable to understand how some data was injected into an object during its instantiation.
 I know what a depency injection manager is but that time was not like the others. It took me one hour to figure where that configuration parameter was coming from. I had to go through 8 code files and 3 configuration files and I had to make several full text searches in order to jump from file to file because following the code was misleading. This exercise was really hard to me because I'm barely able to remember my own phone number so I had to write down to paper the path of your clever way to do depency injection because I couldn't keep it in memory.
```

Comment:
"This exercise was really hard to me because I'm barely able to remember my own phone number so I had to write down to paper the path of your clever way to do depency injection because I couldn't keep it in memory. "
=> phrase un peu longue, à couper si possible?